src/class-wpml-elementor-translatable-nodes.php

```diff
@@ -481,6 +481,11 @@ public function initialize_nodes_to_translate() {
 				'fields'            => array(),
 				'integration-class' => 'WPML_Elementor_Accordion',
 			),
+			'testimonial-carousel'   => array(
```

Comment:
Don't you add a test case for this @adrianosferreira? (like in https://github.com/OnTheGoSystems/wpml-page-builders-elementor/pull/20/files#diff-5a8eacc07e8d4c088de4cbf066fe61fdR236)